src/asciidoc/docs/guidelines/developerTip.PersistingDataAcrossSessions.adoc

```diff
@@ -0,0 +1,50 @@
+= Developer Tip: Persisting data across sessions
+:page-slug: persisting-data-across-sessions
+:page-description: How to persist data that doesn't expire with the session.
+
+Data can be stored temporarily using window.sessionStorage. However, data stored in this way will expire when the session ends. If data needs to be persisted across sessions, `window.localStorage` may be used. Data stored in this way has no expiration date. It will be available until the browser's cache is cleared.
+
+== Using local storage
+There are two ways to use `window.localStorage`.
+
+* Property names may used directly:
++
+[source,json]
+----
+localStorage.username = 'johnsmith001';
+console.log(localStorage.username); // prints johnsmith001 to the console
+delete localStorage.username;
+console.log(localStorage.username); // prints undefined to the console
+----
+
+* Alternatively, `localStorage` provides the methods `setItem` and `removeItem`, which may be used as follows:
++
+[source,json]
+----
+localStorage.setItem('username', 'johnsmith001');
+console.log(localStorage.getItem('username')); // prints johnsmith001 to the console
+localStorage.removeItem('username');
+console.log(localStorage.getItem('username')); // prints undefined to the console
+----
+
+== Things to look out for
```

Comment:
What do you think of using "Developer tips" as the section heading here?

Reply:
Sounds good